core-services/src/test/java/org/zalando/nakadi/service/publishing/NakadiKpiPublisherTest.java

```diff
@@ -85,7 +86,7 @@ public void testPublishingAccessLogWithAvro() throws IOException {
         final var avroSchema = new AvroSchema(new AvroMapper(), new ObjectMapper(), eventTypeRes);
         final NakadiKpiPublisher publisher = new NakadiKpiPublisher(featureToggleService,
                 jsonProcessor, binaryProcessor, usernameHasher,
-                new EventMetadataTestStub(), new UUIDGenerator(), avroSchema);
+                new EventMetadata(new UUIDGenerator()), new UUIDGenerator(), avroSchema);
```

Comment:
Not sure why `UUIDGenerator` two times?

Reply:
Good catch, I'll fix that.
It is in test though.